lib/pure/unittest.nim

```diff
@@ -226,6 +226,9 @@ template skip* =
   testStatusIMPL = SKIPPED
   checkpoints = @[]
 
+macro isLiteral(value: expr[typed]): expr =
+  bool(value.kind == nnkSym and value.symbol.getImpl.kind == nnkNilLit)
```

Comment:
* Conversion to bool is unnecessary.
* isLiteral seems to be a misnomer and why does it only check for symbolic constants?